platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -53,7 +53,6 @@
 		3510FFEC1D6D9C7A00F413B2 /* NSComparisonPredicate+MGLAdditions.mm in Sources */ = {isa = PBXBuildFile; fileRef = 3510FFE91D6D9C7A00F413B2 /* NSComparisonPredicate+MGLAdditions.mm */; };
 		3510FFED1D6D9C7A00F413B2 /* NSComparisonPredicate+MGLAdditions.mm in Sources */ = {isa = PBXBuildFile; fileRef = 3510FFE91D6D9C7A00F413B2 /* NSComparisonPredicate+MGLAdditions.mm */; };
 		3510FFF01D6D9D8C00F413B2 /* NSExpression+MGLAdditions.h in Headers */ = {isa = PBXBuildFile; fileRef = 3510FFEE1D6D9D8C00F413B2 /* NSExpression+MGLAdditions.h */; settings = {ATTRIBUTES = (Public, ); }; };
-		3510FFF11D6D9D8C00F413B2 /* NSExpression+MGLAdditions.h in Headers */ = {isa = PBXBuildFile; fileRef = 3510FFEE1D6D9D8C00F413B2 /* NSExpression+MGLAdditions.h */; settings = {ATTRIBUTES = (Public, ); }; };
```

Comment:
I don't know why this file keeps popping out. Just make sure it's public.

Reply:
Even now, it seems to exist twice:

![screen shot 2018-04-25 at 2 34 56 pm](https://user-images.githubusercontent.com/1198851/39265677-e3a741e8-4895-11e8-86f4-523741929ce7.png)

... will take another look.

Reply:
This should be taken care of in: https://github.com/mapbox/mapbox-gl-native/pull/11771/commits/650adae9fc3f3c73c6de446ebf049c99ca1abbf3